target/linux/mediatek/filogic/base-files/etc/board.d/02_network

```diff
@@ -67,6 +67,9 @@ mediatek_setup_interfaces()
 	xiaomi,redmi-router-ax6000-ubootmod)
 		ucidef_set_interfaces_lan_wan "lan2 lan3 lan4" wan
 		;;
+	tplink,ex820v)
+		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3 lan4 lan6" "eth1 wan"
```

Comment:
Why do you list 7 port here, but the device only has 5 ports?

Reply:
Thank you @hauke, this is now fixed.